sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminAsyncClient.java

```diff
@@ -0,0 +1,611 @@
+    private final ClientLogger logger = new ClientLogger(PersonalizerAdminAsyncClient.class);
+
+    PersonalizerAdminAsyncClient(PersonalizerClientV1Preview3Impl service) {
+        this.service = service;
+    }
+
+    /**
+     * Submit a new Offline Evaluation job.
+     * @param evaluationOptions The Offline Evaluation job definition.
+     * @return a counterfactual evaluation on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<PersonalizerEvaluation> createEvaluation(PersonalizerEvaluationOptions evaluationOptions) {
+        return createEvaluationWithResponse(evaluationOptions).flatMap(FluxUtil::toMono);
+    }
+
+    /**
+     * Submit a new Offline Evaluation job.
+     * @param evaluationOptions The Offline Evaluation job definition.
+     * @return a counterfactual evaluation along with {@link ResponseBase} on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<PersonalizerEvaluation>> createEvaluationWithResponse(PersonalizerEvaluationOptions evaluationOptions) {
+        try {
+            return withContext(context -> createEvaluationWithResponse(evaluationOptions, context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Submit a new Offline Evaluation job.
+     * @param evaluationOptions The Offline Evaluation job definition.
+     * @param context The context to associate with this operation.
+     * @return a counterfactual evaluation along with {@link ResponseBase} on successful completion of {@link Mono}.
+     */
+    Mono<Response<PersonalizerEvaluation>> createEvaluationWithResponse(PersonalizerEvaluationOptions evaluationOptions, Context context) {
+        Objects.requireNonNull(evaluationOptions, "'evaluationContract' is required and can not be null.");
+        return service.getEvaluations().createWithResponseAsync(evaluationOptions, context)
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(response -> new SimpleResponse<>(response, response.getValue()));
+    }
+
+    /**
+     * Get the Offline Evaluation associated with the Id.
+     * @param evaluationId Id of the Offline Evaluation.
+     * @return the Offline Evaluation associated with the Id on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<PersonalizerEvaluation> getEvaluation(String evaluationId) {
+        return getEvaluationWithResponse(evaluationId).flatMap(FluxUtil::toMono);
+    }
+
+    /**
+     * Get the Offline Evaluation associated with the Id.
+     * @param evaluationId Id of the Offline Evaluation.
+     * @return the Offline Evaluation associated with the Id along with {@link Response} on successful completion of
+     * {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
```

Comment:
Need to add @throws for the IllegalArgumentException that is being thrown in javadocs.